guide/14-deep-learning/doccano setup guide.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<blockquote>Navigate to doccano_deployment_scriptfolder.</blockquote><blockquote>Run install.bat as administrator. Please be patient, the setup will take ~40 minutes to complete.</blockquote><blockquote>On the command prompt, you will be asked to create your username and password for accessing Doccano.</blockquote><blockquote>Once the command prompt closes, you should have Doccano running on your local system.</blockquote><p> </p><p>Suggestion:</p><p> </p><p>I second atishay's general suggestions. apart from that, If my 'readme.md' edits are included,</p><p>then these lines can be aligned with those edits or removed to reduce redundant info.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>